diff --git i/GUISupport/QtWebkit/CMakeLists.txt w/GUISupport/QtWebkit/CMakeLists.txt
index 356327b..bbd3a93 100644
--- i/GUISupport/QtWebkit/CMakeLists.txt
+++ w/GUISupport/QtWebkit/CMakeLists.txt
@@ -7,16 +7,21 @@ set(LibSrcs ${QVTKLibSrcs} vtkQtRichTextView.cxx)
 set(MocHeaders ${QVTKMocHeaders} vtkQtRichTextView.h)
 
 if(VTK_QT_VERSION VERSION_GREATER "4")
-  find_package(Qt5WebKitWidgets REQUIRED QUIET)
-  include_directories(${Qt5WebKitWidgets_INCLUDE_DIRS})
-  add_definitions(${Qt5WebKitWidgets_DEFINITIONS})
+  find_package(Qt5 COMPONENTS WebKitWidgets QUIET)
+  if(Qt5WebKitWidgets_FOUND)
+    include_directories(${Qt5WebKitWidgets_INCLUDE_DIRS})
+    add_definitions(${Qt5WebKitWidgets_DEFINITIONS})
 
-  qt5_wrap_ui(UI_FILES vtkQtRichTextView.ui)
-  qt5_wrap_cpp(LibMocSrcs ${MocHeaders})
+    qt5_wrap_ui(UI_FILES vtkQtRichTextView.ui)
+    qt5_wrap_cpp(LibMocSrcs ${MocHeaders})
 
-  set(QT_LIBRARIES ${Qt5WebKitWidgets_LIBRARIES})
+    set(QT_LIBRARIES ${Qt5WebKitWidgets_LIBRARIES})
+    set(_FOUND 1)
+  else()
+    message(STATUS "Qt5WebKitWidgets not found. vtkQtRichTextView is disabled.")
+  endif()
 else()
-  find_package(Qt4 REQUIRED QtCore QtGui QtWebKit QUIET)
+  find_package(Qt4 QtCore QtGui QtWebKit QUIET)
 
   # import Qt4 build settings
   if(QT_PHONON_FOUND AND APPLE)
@@ -28,9 +33,13 @@ else()
 
   qt4_wrap_ui(UI_FILES vtkQtRichTextView.ui)
   qt4_wrap_cpp(LibMocSrcs ${MocHeaders})
+  if (QT_WEBKIT_FOUND)
+    set(_FOUND 1)
+  endif()
 endif()
 
-set(${vtk-module}_NO_HeaderTest 1)
-vtk_module_library(${vtk-module} ${LibSrcs} ${UI_FILES} ${LibMocSrcs})
-
-target_link_libraries(${vtk-module} LINK_PRIVATE ${QT_LIBRARIES})
+if(${_FOUND})
+  set(${vtk-module}_NO_HeaderTest 1)
+  vtk_module_library(${vtk-module} ${LibSrcs} ${UI_FILES} ${LibMocSrcs})
+  target_link_libraries(${vtk-module} LINK_PRIVATE ${QT_LIBRARIES})
+endif()
